admin/lib/generators/solidus_admin/component/component_generator.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module SolidusAdmin
+  class ComponentGenerator < Rails::Generators::NamedBase
+    source_root File.expand_path('templates', __dir__)
+
+    argument :attributes, type: :array, default: [], banner: "attribute"
+
+    class_option :html, type: :boolean, default: true
```

Comment:
What's the use case to generate the template without html? If it's the inline render, shouldn't we add that block to the generated component?